### ISSUE TITLE
Block state conflict on 50295

### ISSUE THREAD

**Issue by int08h** [ethereum/ethereumj#286]
Using latest develop 389f5af4776997037652422819ecf4ee54e6d758 a block state conflict is logged. This is repeatable, I've set database.reset=true and attempted many times.
```
15:51:04.047 [blockchain]  Block's given Receipt Hash doesn't match: b0ae612e16de5b3d05f5088caeceb3b072f47b6f50e86e214360d89adf459d6c != 8439543af05add88b7c42edd8aa91cccdd9d32ea511fb1545516772674a3e71d
15:51:04.048 [state]  BLOCK: STATE CONFLICT! block: 50295 worldstate 0f00cd5896f87985664524efafd25747839265937aa1e1980f6d1af6d8c7f388 mismatch
CONFLICT: BLOCK #50295
```

**Comment by romanman** [comment]
@int08h : just solved that one , upgrade to the last snapshot of 1.0.0

**Comment by int08h** [comment]
Wow, that was fast! Will do.

**Comment by romanman** [comment]
@int08h : we found that yesterday , but thanks for double checking :+1:

**Comment by int08h** [comment]
LOL, I'm behind the times. ;)

Just pulled, all is well. Thank you.

**Comment by romanman** [comment]
@int08h : you are welcome , and don't worry we left some 
bugs for you to find them first ;-)

**Comment by romanman** [comment]
Fixed in: 389f5af4776997037652422819ecf4ee54e6d758
